packages/rs-platform-serialization-derive/src/derive_bincode.rs

```diff
@@ -59,6 +59,9 @@ pub(crate) fn derive_decode_inner(input: TokenStream) -> Result<TokenStream> {
     generator.finish()
 }
 
+#[allow(dead_code)]
+#[deprecated(note = "This function is marked as unused.")]
+#[allow(deprecated)]
```

Comment:
Why do we allow deprecated? It's using some deprecated stuff?

Reply:
A decent bit of the "unused" stuff was only used in other unused stuff; as a result, if I simply mark function as unused and deprecated; then the other unused function that technically used it, would throw a new warning for using deprecated code; as such; functions found to be unused; for consistency are given all three; dead code, deprecated and allow deprecated